GitUtils.ps1

```diff
@@ -287,15 +281,34 @@ function Find-Pageant() {
 
 # Attempt to guess $program's location. For ssh-agent/ssh-add.
 function Find-Ssh($program = 'ssh-agent') {
+    $sshLocation = Get-Command $program -TotalCount 1 -ErrorAction SilentlyContinue
+    if($sshLocation){
+        return $sshLocation
+    }
+
     Write-Verbose "$program not in path. Trying to guess location."
     $gitItem = Get-Command git -Erroraction SilentlyContinue | Get-Item
     if ($gitItem -eq $null) { Write-Warning 'git not in path'; return }
 
     $sshLocation = join-path $gitItem.directory.parent.fullname bin/$program
-    if (get-command $sshLocation -Erroraction SilentlyContinue) { return $sshLocation }
+    if (get-command $sshLocation -Erroraction SilentlyContinue) { 
+        return $sshLocation 
+    }
 
     $sshLocation = join-path $gitItem.directory.parent.fullname usr/bin/$program
-    if (get-command $sshLocation -Erroraction SilentlyContinue) { return $sshLocation }
+    if (get-command $sshLocation -Erroraction SilentlyContinue) { 
+        return $sshLocation 
+    }
+}
+
+function Find-Service($service = 'ssh-agent') {
+     # Find if wind32-openssh available and use it
+    return Get-Service -Name $service -ErrorAction SilentlyContinue

```

Comment:
All references to Get-Service and Start-Service need to be behind a "Windows-only" firewall.